content/applications/hr/time_off.rst

```diff
@@ -632,114 +741,169 @@ Dashboard
+A new allocation request can be made from this view, as well. Click the :guilabel:`New` button to
 :ref:`request an allocation <time_off/request-allocation>`.
 
+.. _time_off/reporting:
+
 Reporting
 =========
 
 The reporting feature allows users to view time off for their team, either by employee or type of
 time off. This allows users to see which employees are taking time off, how much time off they are
 taking, and what time off types are being used.
 
+Any report can be added to a spreadsheet through the :guilabel:`Insert in Spreadsheet` button that
+appears in the top-left of a report. If the *Documents* application is installed, an option to add
+the report to a spreadsheet appears. If not, the report can be added to a *Dashboard*.
+
 By employee
 -----------
 
 To view a report of employee time off requests, navigate to :menuselection:`Time Off app -->
 Reporting --> by Employee`.
 
-The default report is a stacked bar chart with the filters of :guilabel:`Active Employee` and
-:guilabel:`Type` in place.
+The default report presents the current year's data in a list view, displaying all the employees in
+alphabetical order. Each employee's line is collapsed by default. To expand a line, click anywhere
+on the line. The view expands and has the time off requests organized by time off type. Click
+anywhere on a time off type line to expand it, and view all the individual time off requests that
+fall under that type.
+
+The information shown in the list includes the :guilabel:`Employee` name, :guilabel:`Number of Days`
+off requested, the :guilabel:`Start Date`, :guilabel:`End Date`, :guilabel:`Status`, and
+:guilabel:`Description`.
 
-Each employee is displayed in their own column, with the bar displaying how many days of each type
-of time off type they requested.
+.. image:: time_off/employee-report.png
+   :align: center
+   :alt: Report of time off, shown by each employee in a list view.
 
-The report can be displayed in other ways. Click the various options at the top of the report to
-view the data differently.
+The report can be displayed in other ways. Click the corresponding button option in the top-right
+corner of the page to view the data in that way. The various options are a :guilabel:`List` (the
+default view) :guilabel:`Graph`, :guilabel:`Pivot` table, or :guilabel:`Calendar` view.
+
+.. image:: time_off/report-options.png
+   :align: center
+   :alt: The various other views highlighted, for the employee time off report.
+
+When a selection has been made, additional options appear for that particular selection.
 
 The graph options are :guilabel:`Bar Chart`, :guilabel:`Line Chart`, or :guilabel:`Pie Chart`. The
 :guilabel:`Bar Chart` includes an option to present the data :guilabel:`Stacked`. Both the
 :guilabel:`Bar Chart` and :guilabel:`Line Chart` have options to present the data in either
 :guilabel:`Descending` or :guilabel:`Ascending` order.
```

Comment:
There is also no mention of the 'Measures' drop-down menu -- nor is there any other mention of the other view specific options...so can either leave that description as-is in line 875, or add the necessary explanations, if you decide to go that route. Your call, though.

Reply:
I opted for a mix- I removed the last detailed chunk and added a link to the new reporting doc that cover all of those buttons and various options.